tests/vm_scheduling/test_vm_scheduling_scenario.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import unittest
+
+from maro.data_lib import BinaryConverter
+from maro.simulator.scenarios.vm_scheduling import CpuReader
+
+
+class CpuReaderTest(unittest.TestCase):
```

Comment:
Currently, the problem found in CpuReader is not handled in this unit test:

You can design a test case to cover the "start_tick" not in the first file.

Also to test the correctness of ``reset()``, you can:
- test like the current implementation
- reset the cpureader
- test again


### Why reset() is important
Unlike the examples we implemented now, which only require the environment to be initialized and run only once. For the training of Reinforcement Learning(RL) agents, the environment needs to be reset again and again (to run the same topology again and again), each time it's called an ``episode``.